cmd/cloudflared/access/cmd.go

```diff
@@ -39,13 +40,22 @@ Add to your {{.Home}}/.ssh/config:
 
 Host {{.Hostname}}
 {{- if .ShortLivedCerts}}
+	{{- if eq .OS "windows"}}
+	ProxyCommand {{.Cloudflared}} access ssh-gen --hostname %h && ssh -tt %r@cfpipe-{{.Hostname}} >&2 <&1
+	{{- else}}
   ProxyCommand bash -c '{{.Cloudflared}} access ssh-gen --hostname %h; ssh -tt %r@cfpipe-{{.Hostname}} >&2 <&1'
+	{{end}}
 
 Host cfpipe-{{.Hostname}}
   HostName {{.Hostname}}
   ProxyCommand {{.Cloudflared}} access ssh --hostname %h
-  IdentityFile ~/.cloudflared/{{.Hostname}}-cf_key
+	{{- if eq .OS "windows"}}
+  IdentityFile {{.CurrWorkingDir}}/{{.Hostname}}-cf_key
+  CertificateFile {{.CurrWorkingDir}}/{{.Hostname}}-cf_key-cert.pub
+	{{- else}}
+	IdentityFile ~/.cloudflared/{{.Hostname}}-cf_key
   CertificateFile ~/.cloudflared/{{.Hostname}}-cf_key-cert.pub
+	{{end}}
 {{- else}}
   ProxyCommand {{.Cloudflared}} access ssh --hostname %h
 {{end}}
```

Comment:
```suggestion
{{- if eq .OS "windows"}}  IdentityFile {{.CurrWorkingDir}}/{{.Hostname}}-cf_key
  CertificateFile {{.CurrWorkingDir}}/{{.Hostname}}-cf_key-cert.pub
{{- else}}  IdentityFile ~/.cloudflared/{{.Hostname}}-cf_key
  CertificateFile ~/.cloudflared/{{.Hostname}}-cf_key-cert.pub
{{end}}
{{- else}}
  ProxyCommand {{.Cloudflared}} access ssh --hostname %h
{{end}}
```

Indentation is weird with go templates, we can't add whitespaces before template statements, and lets also replace tabs with double withspaces